src/control/settings/ButtonConfig.cpp

```diff
@@ -20,12 +20,34 @@ auto ButtonConfig::getDrawingType() -> DrawingType { return this->drawingType; }
 
 auto ButtonConfig::getAction() -> ToolType { return this->action; }
 
-void ButtonConfig::acceptActions(ToolHandler* toolHandler) {
+void ButtonConfig::initButton(ToolHandler* toolHandler, Buttons button) {
```

Comment:
Shouldn't that class be called `Button`?

Reply:
Agreed seems also more natural to me.

Reply:
Fixed in 38d509654dd5b017f84bf8b17f59b0a907d83638. Please resolve if you agree with the solution.